<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupPage</class>
 <widget class="QWidget" name="GroupPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Appearance</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Override NPC color:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="buddy">
         <cstring>npcOverrideColorPushButton</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="npcOverrideColorCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Your color:</string>
        </property>
        <property name="buddy">
         <cstring>yourColorPushButton</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="yourColorPushButton">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="npcOverrideColorPushButton">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Filtering and Order</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="0">
       <widget class="QCheckBox" name="showTokensCheckbox">
        <property name="text">
         <string>Show Character Images</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="npcHideCheckbox">
        <property name="text">
         <string>Hide NPCs</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="npcSortBottomCheckbox">
        <property name="text">
         <string>Sort NPCs to bottom</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QComboBox" name="tokenSizeComboBox">
        <item>
         <property name="text">
          <string>16 px</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>32 px</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>48 px</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>64 px</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Character Image Size</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>npcSortBottomCheckbox</tabstop>
  <tabstop>npcOverrideColorPushButton</tabstop>
  <tabstop>npcOverrideColorCheckBox</tabstop>
  <tabstop>npcHideCheckbox</tabstop>
  <tabstop>yourColorPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
